caikit_nlp/toolkit/text_generation/tgis_utils.py

```diff
@@ -40,6 +40,16 @@
     preserve_input_text: str
         Whether or not the source string should be contained in the generated output,
         e.g., as a prefix.
+    input_tokens: str
+        Whether or not to include list of input tokens.  
+    generated_tokens: str
+        Whether or not to include list of individual generated tokens.  
+    token_logprobs: str
+        Whether or not to include logprob for each returned token.  
+        Applicable only if generated_tokens == true and/or input_tokens == true
+    token_ranks: str
```

Comment:
Based on the module runtime parameters these all appear to be `bool` not `str`? (Seems `preserve_input_text` might not have been accurate either, if that could also be updated here)